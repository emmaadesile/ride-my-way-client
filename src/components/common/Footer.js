import React from 'react';

const Footer = () => {
  return (
    <div>
      <footer className="footer ">
        <p>&copy; 2018. Made by Emmanuel</p>
      </footer>
    </div>
  )
}

export default Footer;