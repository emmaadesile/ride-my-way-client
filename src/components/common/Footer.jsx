import React from 'react';

const Footer = () => {
  return (
    <footer className="footer ">
    <p>&copy; 2018. Made by Emmanuel</p>
  </footer>
  )
}
export default Footer;