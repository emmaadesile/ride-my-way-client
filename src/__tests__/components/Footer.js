import React from "react";
import { shallow } from "enzyme";
import toJson from 'enzyme-to-json';

import Footer from "../../components/common/Footer";

describe("Footer component test", () => {
  it("renders the footer component", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.exists()).toBe(true);
  });

  it('should render correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
