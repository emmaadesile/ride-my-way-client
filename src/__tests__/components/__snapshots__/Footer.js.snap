// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component test should render correctly 1`] = `
<footer
  className="footer "
>
  <p>
    Â© 2018. Made by Emmanuel
  </p>
</footer>
`;
